.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,37 @@
+Recipes added with this pull request (max 5):
+ 
+ - package1
+ - package2
+ 
+I have checked the following in my recipes:
+* [ ] My files end with only 1 newline character, no more, no less.
+* [ ] My recipe follow the same style as [example](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml) as much as possible.
```

Comment:
We might need to clean that up to match the style we want first. It fits an older acceptable style. Also, we have been discussing having a few, which we should work on somewhere.